ClusterTest/src/main/scala/net/snowflake/spark/snowflake/testsuite/LowMemoryStressSuite.scala

```diff
@@ -76,16 +76,17 @@ class LowMemoryStressSuite extends ClusterTestSuiteBase {
     // Convert RDD to DataFrame
     val df = sparkSession.createDataFrame(testRDD, schema)
 
-    // Write to snowflake
-    df.write
-      .format(TestUtils.SNOWFLAKE_NAME)
-      .options(TestUtils.sfOptionsNoTable)
-      .option("dbtable", test_big_partition)
-      .mode(SaveMode.Overwrite)
-      .save()
-
-    log.info(
-      s"""Finished the first multi-part upload test ======================""".stripMargin)
+    // Below test code can be enabled only if AWS multiple part upload API is used.
+//    // Write to snowflake
+//    df.write
+//      .format(TestUtils.SNOWFLAKE_NAME)
+//      .options(TestUtils.sfOptionsNoTable)
+//      .option("dbtable", test_big_partition)
+//      .mode(SaveMode.Overwrite)
+//      .save()
+//
+//    log.info(
+//      s"""Finished the first multi-part upload test ======================""".stripMargin)
```

Comment:
can we remove this commented off code?

Reply:
I prefer to keep it.
The feature is disabled for SC 2.8.2 release.The feature will be re-enabled in the future. So, the test code needs to be brought back too. So it should be better to keep it. Does it make sense?